lib/Screens/UserScreen/Client/client_screen.dart

```diff
@@ -20,70 +22,54 @@ class ClientScreen extends ConsumerWidget {
       ),
       body: Padding(
         padding: const EdgeInsets.only(top: 30, bottom: 50),
-        child: ListView.builder(
-          itemCount: clients.length,
-          itemBuilder: (BuildContext context, int index) { 
-            final client = clients[index];
-            return Padding(
-              padding: const EdgeInsets.symmetric(vertical: 15, horizontal: 25),
-              child: ListTile(
-                onTap: () {
-                  // Acción al hacer clic en un cliente
+        child: usersGet.when(
+          data: (users) {
+            if (users.isEmpty) {
+              return const Center(
+                child: Text('No hay usuarios en este momento'),
+              );
+            } else {
+              return ListView.builder(
+                itemCount: users.length,
+                itemBuilder: (BuildContext context, int index) {
+                  final UserEntity user = users[index];
+                  return Padding(
+                    padding: const EdgeInsets.symmetric(
+                        vertical: 15, horizontal: 25),
+                    child: ListTile(
+                      onTap: () {
+                        // Acción al hacer clic en un usuario
+                      },
+                      onLongPress: () {
+                        showAlert(context, ref);
+                      },
+                      title: Text(
+                          "Nombre: ${user.name} - Empresa: ACA IRIA LA EMPRESA CON LA RELACION"),
```

Comment:
will itshowed to usres?

Reply:
yes, that's how they had done the view, the related companies would be shown to the users